drivers/hmis/app/models/hmis/form/definition.rb

```diff
@@ -413,6 +413,15 @@ def valid_status_for_submit?
     published? || retired?
   end
 
+  def supports_save_in_progress?
```

Comment:
Makes sense that this is on the definition!
I think this could be slightly optimized by returning immediately if this isn't an assessment definition.
The file search can also be a _little_ more succinct by using link_id_item_hash:
```
    # Only Assessments can be saved as WIP
    return false unless ASSESSMENT_FORM_ROLES.include?(role.to_sym)
    # If the assessment contains a File collection item, it cannot be saved as WIP (# link issue nr?)
    return false if link_id_item_hash.values.find? { |item| item.type == 'FILE' }

    true
```

Reply:
Oh and perhaps including `IMAGE` type as well